Cargo.toml

```diff
@@ -12,7 +12,10 @@ readme = "README.md"
 path = "src/lib.rs"
 
 [dependencies]
-egui = "0.20"
+egui = { version = "0.21", default-features = false }

```

Comment:
Small change here: egui has the `default_fonts` feature by default. It bundles the default egui fonts (hence the name).

Previously, `egui-notify` would implicitly "force" the `default_fonts` feature onto users of `egui-notify`, bloating their binaries if they were not using `default_fonts` themselves.